client/src/pages/LandingPage/useStyles.ts

```diff
@@ -0,0 +1,42 @@
+import { Theme } from '@mui/material';
+import { makeStyles } from '@mui/styles';
+
+export const useStyles = makeStyles((theme: Theme) => ({
+  left: {
+    padding: theme.spacing(15, 15, 0),
```

Comment:
Let's work on the padding based on the breakpoints, smaller screen this is `120px`